locations/spiders/jax_tyres_and_auto_au.py

```diff
@@ -37,7 +37,7 @@ def parse(self, response):
         day_names = response.xpath('//div[contains(@class, "pgStLcSn-openingHours")]/div[1]/p/text()').getall()
         hour_ranges = response.xpath('//div[contains(@class, "pgStLcSn-openingHours")]/div[2]/p/text()').getall()
         for i in range(1, 7, 1):
```

Comment:
FYI `zip` would make this more readable, at least to me, but I don't know if it has any other pros/cons.

```python
        for day, hours in zip(day_names, hour_ranges):
```